piker/clearing/_ems.py

```diff
@@ -20,7 +20,7 @@
 """
 from contextlib import asynccontextmanager
 from dataclasses import dataclass, field
-from math import isnan
+# from math import isnan
```

Comment:
does this still need to be here?

Reply:
i don't think so, i'll scan the old comments to make sure.